extensions/amp-story/1.0/amp-story.css

```diff
@@ -472,6 +472,7 @@ amp-story-grid-layer .i-amphtml-embedded-component::after {
   width: var(--i-amphtml-story-layer-width, 100%);
   height: var(--i-amphtml-story-layer-height, 100%);
   font-size: calc(var(--i-amphtml-story-layer-height, 100vh) / 10);
+  margin-left: calc((var(--story-page-vw, 1%) * 100 - var(--i-amphtml-story-layer-width, 100%)) * 0.5); /* Set margin to 1/2 of page-width - layer-width to center */
```

Comment:
To explain the intended behavior you could add "Centers element".

Reply:
Yes! I think it makes more sense to explain center **layer** rather than **element** but hope it's more clear now